### ISSUE TITLE
pyDGS output vs actual sieve data

### ISSUE THREAD

**Issue by jacobstasiewicz** [dbuscombe-usgs/pyDGS#16]
I ran test.py on pyDGS with ```resolution=1```  ```maxscale=20``` on cropped 512x512 images with the actual sieve phi values for d5, d10, d16, d25, d50, d75, d84, d95  =  ```[-0.2204 0.3029 0.6865 1.1465 1.9058 2.2877 2.4106 2.4925 2.7558]``` switched to mm and flipped to sediment finer the values =```[0.1481 0.1777 0.1881 0.2048 0.2669 0.4517 0.6214 1.1651]``` the pyDGS output values are ```[0.205355221 0.410710441 0.58721403 0.658453122 0.859771063 1.069730853 1.149177598 1.249914063]``` for an image taken 5 inches above the sediment. The matlab script used for the plot is 
```
clear
clc
%%% plotting sieve data vs pyDGS
%actual phi values (phi5 phi16 phi25 phi50 phi75 phi84 phi 90 phi95)
phis =[-0.2204 0.6865 1.1465 1.9058 2.2877 2.4106 2.4925 2.7558];
%flip to finer than (output will be phi5 phi10 phi16 phi25 phi50 phi75
%phi84 phi95)
flipped = flip(phis);
%convert to mm
mm= 2.^-(flipped);
%pyDGS data
pyDGS5in = [0.205355221 0.410710441 0.58721403 0.658453122 0.859771063 1.069730853 1.149177598 1.249914063];
%create figure
figure;
plot(mm, pyDGS5in,'-o'), xlabel('actual "mm"'), ylabel('predicted "mm"'), title('5in'), hold on, plot(mm,mm,'-*'), legend({'Actual vs pyDGS','1:1'},'Location','southeast'), grid on, hold off
```
![5inplot](https://user-images.githubusercontent.com/72474059/121927775-70764480-cd0d-11eb-99b4-b2f69a847c89.jpg)

**Comment by ddbuscombe** [comment]
Do you mean you modified 'test.py' and pointed it to your own data?

If so, there could be a number of reasons why the method overpredicts, including, in order of usual importance:
1. `x` is too large (for sands, typically negative values)
2. maxscale is too small
3. your resolution is badly specified - note that any macro lens you use will change the resolution. Resolution is very difficult to measure - I usually have 3 people make measurements and average them or use the minimum

Converting between mm and pixels is already problematic because of error in resolution gets propagated, but convering from phi to mm to px is especially problematic due to loss of precision due to two arithmetic operations when one should be enough. That's why I dont typically compare to sieves, but why the `x` parameter exists ... 

In short, start by modifying `x`

**Comment by ddbuscombe** [comment]
You'll see sections in all my relevant papers that explain why sieves are a poor comparison to DGS measurements - they measure two different things - hence the development and use of the `x` parameter - I hope that's clear

**Comment by ddbuscombe** [comment]
Finally, never use inches or non-metric units in science
